node/subsystem/src/messages.rs

```diff
@@ -220,12 +220,32 @@ impl BitfieldDistributionMessage {
 	}
 }
 
+/// Bitfield signing message.
+///
+/// Currently non-instantiable.
+#[derive(Debug)]
+pub enum BitfieldSigningMessage {}
+
+impl BitfieldSigningMessage {
+	/// If the current variant contains the relay parent hash, return it.
+	pub fn relay_parent(&self) -> Option<Hash> {
+		None
+	}
+}
+
 /// Availability store subsystem message.
 #[derive(Debug)]
 pub enum AvailabilityStoreMessage {
 	/// Query a `PoV` from the AV store.
 	QueryPoV(Hash, oneshot::Sender<Option<PoV>>),
 
+	/// Query whether a `PoV` exists within the AV Store.
+	///
+	/// This is useful in cases like bitfield signing, when existence
+	/// matters, but we don't want to necessarily pass around multiple
+	/// megabytes of data to get a single bit of information.
+	QueryPoVAvailable(Hash, oneshot::Sender<bool>),
```

Comment:
We could potentially avoid this variant, just using `QueryPoV` instead. Does it create any problems to create this variant?

Reply:
I think this message type has some utility, but it's also not what the bitifeld signing subsystem needs.

Bitfield signing is supposed to query for whether we have our specific _chunk_ of the PoV, not the whole PoV itself.

Quoting from the guide's description of bitfield signing:
> For each bit in the bitfield, if there is a candidate pending availability, query the Availability Store for whether we have the availability chunk for our validator index.